### ISSUE TITLE
Could you please help me with how to modify the default_font setting?

### ISSUE THREAD

**Issue by Nanyuting** [JujuAdams/Scribble#590]
Code Example:
var _string = "[fnt1]Juju is the best {_value},he is powerful"
{_value} will be replaced with "[fnt2]mage[/f]" during actual gameplay.
The expected display should be:
"[fnt1]Juju is the best [fnt2]mage[/f],he is powerful"
However, the actual display results in:
"[fnt1]Juju is the best [fnt2]mage[/f]，? ? ?"
This issue occurs because [fnt1] is a Chinese font, and __scribble_state.__default_font does not support Chinese.

What I have in mind is that I might need to modify __scribble_state.__default_font = "fnt1" before rendering the string, but I'm not sure whether this is possible.

I’ve had difficulty finding similar information online, so I’m afraid I’m bothering you with this. 
Or should I modify the settings elsewhere?
I apologize for the inconvenience.
![c4bb6bd8-3ddc-4cf0-bde1-eb6d8a3e69e0](https://github.com/user-attachments/assets/da7814ee-ef06-4586-9824-f8edab50ba55)

**Comment by JujuAdams** [comment]
You can use [`scribble_font_set_default()`](https://www.jujuadams.com/Scribble/#/latest/fonts?id=scribble_font_set_defaultfontname) to set the default font.

In your case, instead of using `[/f]` I'd use `[fnt1]`.

**Comment by Nanyuting** [comment]
Thank you so much, Juju.
Especially for your suggestion ‘instead of using [/f] I'd use [fnt1],’ which is something I could have thought of. However, when I ran into problems, my thinking got stuck in a dead end, and I couldn't get out of the maze.
I really appreciate your support. 
By studying Scribble's functions, I made my own editor, and I'm using it to remake my game. 
I'll send you good news later.
